spec/vcr/ShippingController/UPS_delivery/behaves_like_a_shipping_carrier/POST_ups_/invalid_params/is_successful.yml

```diff
@@ -0,0 +1,103 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://onlinetools.ups.com/ups.app/xml/Rate
+    body:
+      encoding: UTF-8
+      string: |
+        <?xml version="1.0"?>
+        <AccessRequest>
+          <AccessLicenseNumber>7CE85DED4C9D07AB</AccessLicenseNumber>
```

Comment:
You should have added this to your gitignore because it can expose secrets. Not good that we can see the AccessLicenseNumber for all the files for what VCR returns.